lib/java/com/google/android/material/datepicker/MaterialCalendarGridView.java

```diff
@@ -46,8 +48,10 @@ public MaterialCalendarGridView(Context context, AttributeSet attrs) {
   }
 
   public MaterialCalendarGridView(Context context, AttributeSet attrs, int defStyleAttr) {
-    super(context, attrs, defStyleAttr);
```

Comment:
MaterialCalendarGridView changes can be reverted as the class isn't publicly exposed.